src/sections/CustomButton.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+interface CustomButtonProps {
+  onClick?: () => void;
+  label: string;
+  icon?: string;
+  iconPosition?: "left" | "right";
+  color?: string;
+  disabled?: boolean;
+  borderColor?: string;
+  href?: string; // Optional URL to navigate to
+  target?: string; // Optional target (_blank, _self, etc.)
+}
+
+const CustomButton: React.FC<CustomButtonProps> = ({
+  onClick,
+  label,
+  icon,
+  iconPosition = "left",
+  color = "#1C1C1C",
+  disabled = false,
+  borderColor = "#3B3B3B",
```

Comment:
Instead of asking the user for `color` and `borderColor`, ask them for the type of button they want. This way, if any theme changes occur later, we only need to update the button colors here, and all the buttons will be updated accordingly